parlai/agents/starspace/starspace.py

```diff
@@ -213,7 +213,11 @@ def override_opt(self, new_opt):
 
     def parse(self, text):
         """Convert string to token indices."""
-        return self.dict.txt2vec(text)
+        vec = self.dict.txt2vec(text)
+        if vec == []:
+            vec = [self.dict["_NULL"]]
```

Comment:
did you mean `self.dict[self.dict.null_token]`? or is this a different special token?

Reply:
i guess that's the same thing?

Reply:
that's not the null token we typically use (i.e. by default), and it's user configurable so Emily's suggestion is better.